app/views/layouts/_topbar.html.erb

```diff
@@ -1,4 +1,11 @@
-        <ul class="loginbar pull-right">
+<ul class="loginbar pull-right">
+  <%if user_signed_in? && current_user.admin?%>
+    <% commit = `git rev-parse HEAD`[0,6] %>
```

Comment:
I know this probably should not be in the view, but where should I add this? In the menus controller?